### ISSUE TITLE
Unable to set gatling.data.writers from system property

### ISSUE THREAD

**Issue by afajl** [gatling/gatling#1099]
First: thanks for an elegant framework!

Easiest way to reproduce:

```
h@p:~/downloads/gatling-charts-highcharts-2.0.0-M1$ export JAVA_OPTS="-Dgatling.data.writers='[console,graphite]'"
h@p:~/downloads/gatling-charts-highcharts-2.0.0-M1$ ./bin/gatling.sh
GATLING_HOME is set to /Users/p/downloads/gatling-charts-highcharts-2.0.0-M1
Exception in thread "main" com.typesafe.config.ConfigException$WrongType: system properties: gatling.data.writers has type STRING rather than LIST
    at com.typesafe.config.impl.SimpleConfig.findKey(SimpleConfig.java:124)
    at com.typesafe.config.impl.SimpleConfig.find(SimpleConfig.java:136)
    at com.typesafe.config.impl.SimpleConfig.find(SimpleConfig.java:142)
    at com.typesafe.config.impl.SimpleConfig.find(SimpleConfig.java:142)
    at com.typesafe.config.impl.SimpleConfig.find(SimpleConfig.java:150)
    at com.typesafe.config.impl.SimpleConfig.find(SimpleConfig.java:155)
    at com.typesafe.config.impl.SimpleConfig.getList(SimpleConfig.java:203)
    at com.typesafe.config.impl.SimpleConfig.getHomogeneousUnwrappedList(SimpleConfig.java:260)
    at com.typesafe.config.impl.SimpleConfig.getStringList(SimpleConfig.java:318)
    at io.gatling.core.config.GatlingConfiguration$.setUp(GatlingConfiguration.scala:118)
    at io.gatling.app.Gatling$.fromMap(Gatling.scala:57)
    at io.gatling.app.Gatling$.runGatling(Gatling.scala:78)
    at io.gatling.app.Gatling$.main(Gatling.scala:53)
    at io.gatling.app.Gatling.main(Gatling.scala)
```

**Comment by gempesaw** [comment]
Ah, I got here trying to use `writers = [console, file, graphite]`, which is apparently the old way, but gives a similar enough error message that Google directs to this thread. FWIW, the current way in 2.1.4 is in the official docs now:

http://gatling.io/docs/2.1.4/realtime_monitoring/index.html#gatling-configuration

```
data {
  writers = "console, file, graphite"
}
```

**Comment by slandelle** [comment]
@gempesaw Fixed, thanks for reporting! #2590
